modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowUsecase.testcase

```diff
@@ -796,6 +796,114 @@ definition {
 			wikiPageTitle = "Wiki Page Title");
 	}
 
+	@description = "LPS-193788: Verify if it is not possible to publish a workflow at a virtual instance when the configuration is disabled at main instance"
+	@priority = 3
+	test CanNotPublishWorkflowAtVirtualInstance {
+		property test.liferay.virtual.instance = "false";
+
+		task ("Given administrators publish and edit workflows configuration is disabled") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "System Settings");
+
+			SystemSettings.gotoConfiguration(
+				configurationCategory = "Workflow",
+				configurationName = "Workflow Definition",
+				configurationScope = "System Scope");
+
+			SystemSettings.configureSystemSetting(
+				enableSetting = "false",
+				settingFieldName = "Allow Administrators to Publish and Edit Workflows");
```

Comment:
@Luizcarlosqueiroz maybe for this one you can just assert it is unchecked, so that way we can cover that it is the default option. What do you think?

Reply:
Makes sense, considering the default value.